src/ray/gcs/gcs_server/gcs_placement_group_scheduler.cc

```diff
@@ -296,27 +293,13 @@ void GcsPlacementGroupScheduler::CommitAllBundles(
                                       node_id,
                                       schedule_failure_handler,
                                       schedule_success_handler](const Status &status) {
+      auto commited_bundle_locations = std::make_shared<BundleLocations>();
       for (const auto &bundle : bundles_per_node) {
         lease_status_tracker->MarkCommitRequestReturned(node_id, bundle, status);
-
-        auto resources = bundle->GetFormattedResources();
-        // For gcs actor scheduler, placement group's wildcard resources have to be
-        // updated incrementally.
-        // TODO(Chong-Li): This part should be removed when PG scheduling is refactored.
-        if (RayConfig::instance().gcs_actor_scheduling_enabled()) {
-          auto wildcard_resources = bundle->GetWildcardResources();
-          for (const auto &resource_entry : wildcard_resources) {
-            auto capacity = gcs_resource_manager_
-                                .GetNodeResources(scheduling::NodeID(node_id.Binary()))
-                                .total.Get(scheduling::ResourceID(resource_entry.first))
-                                .Double();
-            resources[resource_entry.first] += capacity;
-          }
-        }
-        // Update the resource in gcs resource manager
-        gcs_resource_manager_.UpdateResources(node_id, resources);
```

Comment:
NOTE: The `UpdateResources` only do two things, one is update the `cluster_resource_manager_` and the another is flush the updated resources to the storage. The first one could be replaced by `CommitBundleResources` and the second one is unnecessary as it could be replayed by placement group when GCS restarts.

Reply:
Glad we are removing the storage things here :)!